Update readme (#29)
esl/mongooseim-docker: README.md

The member container can be created with the following command

```
docker run -t -d -P -h mongooseim-1 --name mongooseim-1  mongooseim
```

After `docker logs mongooseim-1` shows something similar to:

```
...
MongooseIM cluster primary node mongooseim@myproject-mongooseim-1
Clustered mongooseim@myproject-mongooseim-1 with mongooseim@myproject-mongooseim-1
Exec: /member/mongooseim/erts-6.3/bin/erlexec -boot /member/mongooseim/releases//mongooseim -embedded -config /member/mongooseim/etc/app.config -args_file /member/mongooseim/etc/vm.args -- live --noshell -noinput +Bd -mnesia dir "/member/mongooseim/Mnesia.mongooseim@myproject-mongooseim-1"
Root: /member/mongooseim
13:58:03.167 [info] msg: "Starting reporters with []\n", options: []
13:58:03.334 [notice] Changed loglevel of /var/log/mongooseim/ejabberd.log to info
13:58:03.357 [info] Application mnesia exited with reason: stopped
13:58:03.752 [notice] Changed loglevel of /var/log/mongooseim/ejabberd.log to warning
```

We can health-check the MongooseIM node with `telnet`.

> Supply the IP based on your setup (usually 127.0.0.1) and port which translates to the container's 5222.

Reviewer: ```suggestion
To do that, you need to provide the IP of the container (usually 127.0.0.1) and the published TCP port which translates to container’s port 5222.
???
```

> In order to find the port you can use the following docker command

Reviewer: ```suggestion
In order to find the port you can use the following docker command:
```

```
docker ps -f "name=mongooseim-1" --format "{{.Names}}: {{.Ports}}"
mongooseim-1: 0.0.0.0:32772->4369/tcp, 0.0.0.0:32771->5222/tcp, 0.0.0.0:32770->5269/tcp, 0.0.0.0:32769->5280/tcp, 0.0.0.0:32768->9100/tcp
```

In the example above you can see that port 5222 inside the container was published on port 32771 on the docker host machine.

> It can be used to check the server is really listening on that port:

Reviewer: ```suggestion
It can be used to check if the server is really listening on that port:
```

which can be found in MongooseIM's repo in `priv/pgsql.sql`

Don't forget to tweak your `mongooseim.cfg` to connect with the services you set up!

> Checkout documentation at https://mongooseim.readthedocs.io/en/latest/advanced-configuration/outgoing-connections/

Reviewer: ```suggestion
See the documentation at https://mongooseim.readthedocs.io/en/latest/advanced-configuration/outgoing-connections/
```
